### PR TITLE
playlist and script documentation

### PR DESCRIPTION
**Pull request by harshmandalgi**

### CODE REVIEW

File: playlist.js [hunk 1]
```diff
@@ -17,25 +18,36 @@ request.onreadystatechange = function () {
...
   videoPlayer.onended = function () {
     if (scorePlayer)
       snackNotif(noLaugh[parseInt(Math.random() * noLaugh.length)]);
     if (i < fileArray.length-1) {
       i++;
-      videoPlayer.src = boketLink+fileArray[i]+'.mp4';
+      // fetching videos from s3 link
+      //videoPlayer.src = boketLink+fileArray[i]+'.mp4';
+      // fetching videos directly from array element text, in this case we are feeding URLs directly into playlist file
+      videoPlayer.src = fileArray[i];
```
Comment (mrgreen-creates) [inline playlist.js]: any reason why do we increment the count before getting the fileArray?
if its safe can we first get the value from fileArray and then increment i after line number 50 ?

File: playlist.js [hunk 2]
```diff
@@ -17,25 +18,36 @@ request.onreadystatechange = function () {
...
+  // wait for the below time in ms to prevent scoring at the start of an video
+  // user may be smiling after previous video ends so we give a window of below mentioned time in which the score will not increase
   setTimeout(function() {
     scorePlayer = true;
   }, 3000);
   var videoPlayer = document.getElementById('challenge');
-  videoPlayer.src = boketLink+fileArray[0]+'.mp4';
+  // fetching videos from s3 link
+  // videoPlayer.src = boketLink+fileArray[0]+'.mp4';
+  // fetching videos directly from array element text, in this case we are feeding URLs directly into playlist file
```
Comment (mrgreen-creates) [inline playlist.js]: remove out the commented out code

**Review by VijayantSoni — APPROVED** [review]
While merging please resolve conflicts and make sure we don't overwrite anything in rel/2.0.
Simplest way, choose `--theirs` strategy while merging.

**Comment by harshmandalgi** [comment]
> While merging please resolve conflicts and make sure we don't overwrite anything in rel/2.0.
> Simplest way, choose `--theirs` strategy while merging.

ok thanks

**Comment by mrgreen-creates** [comment]
Please resolve the conflicts and merge it and make sure we don't use `snackNotif` but use `showNotificationAlert` method when displaying information related to players scores

**Pull request closed without merging**